package/skelcd-control-SLE_RT.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Wed Mar 25 02:01:24 UTC 2020 - Jeffrey Cheung <jcheung@suse.com>
+
+- Bump to 15.2.2
```

Comment:
please merge those two request together. So it contain spec file change and also changelog update and to auto submit we need also bugzilla ID, so please reuse the previous one for this change.

Thanks